e2e/testcases/correction-birth/correct-birth-record-3.spec.ts

```diff
@@ -299,17 +304,14 @@ test.describe.serial(' Correct record - 3', () => {
 
         await expect(
           page.locator('#mother-content #Date').getByRole('deletion')
-        ).toHaveText(
-          format(parseISO(declaration.mother.birthDate), 'dd MMMM yyyy'),
-          { ignoreCase: true }
-        )
+        ).toHaveText(formatDateTo_ddMMMMyyyy(declaration.mother.birthDate), {
```

Comment:
Is ignore case needed? If I read right, we have just formatted the text to our liking? A minor thing that seems to reoccur throughout.

On the otherhand introducing `_ddMMMMyyyy` is nice, even though it breaks the camelCasing pattern. And I understand why.

Other way we could limit usage to 'right' ones but allow extending the same function further down the road

```
const DayFormat = {
  ddMMMMyyyy: 'dd MMMM yyyy',
  yyyyMMdd: 'yyyy-MM-dd',
} as const

const dayFormats = Object.values(DayFormat)

export type DayFormat = (typeof dayFormats)[number]

// DayFormat can now act as a type and value. A better enum essentially.
export const formatDate = (date: string, format: DayFormat) =>
  format(parseISO(date), format)
}

formatDateTo('2021-01-01', DayFormat.ddMMMMyyyy) // '01 January 2021'?
formatDateTo('2021-01-01', DayFormat.yyyyMMdd) // '2021-01-01'?